src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/NativeTypeTableFilter.java

```diff
@@ -122,6 +121,6 @@ public <StmtT extends OngoingWhereClause<StmtT>> StmtT apply(
     }
 
     return ongoingWhereClause.where(
-        Relation.column(path).build(operator.predicate.cql, bindMarker()));
+        Relation.column(getPathAsCqlIdentifier()).build(operator.predicate.cql, bindMarker()));
```

Comment:
Another important fix: instead of passing column name as `String` needs to pass as `CqlIdentifier` to retain casing etc.